src/components/journey-header/journey-header.ts

```diff
@@ -47,7 +47,7 @@ export class SbbJourneyHeaderElement extends SbbNegativeMixin(LitElement) {
       <sbb-title
         level=${this.level || nothing}
         ?negative=${this.negative}
-        visual-level=${this.size === 'l' ? '4' : '5'}
+        visual-level=${this.size === 'l' ? '4' : this.size === 'm' ? '5' : '6'}
```

Comment:
Completely fine, I was only thinking, maybe a map (size to level) could be clearer?